### PR TITLE
Add retry library

### PR DESCRIPTION
**Pull request by anyasabo**
Adds a shared library that overrides the default checkout behavior to include a sleep+retry, which should help with our checkout/github connection failures. Related PR is needed to add this library to the devops-ci jenkins

### CODE REVIEW

**Comment by anyasabo** [comment]
jenkins test this please

**Comment by anyasabo** [comment]
jenkins test this please

**Comment by anyasabo** [comment]
The library has been added to the jenkins server so we should be good to use this. It will hopefully make our CI runs a little more stable.

File: .ci/pipelines/build.Jenkinsfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@Library('apm@current') _
```
Comment (sebgl) [inline .ci/pipelines/build.Jenkinsfile]: can we add a comment to explain what this does?

**Pull request merged**